src/Sylius/Behat/Client/ApiPlatformClient.php

```diff
@@ -94,6 +94,17 @@ public function delete(string $resource, string $id): void
         $this->request('DELETE', sprintf('/new-api/%s/%s', $resource, $id), []);
     }
 
+    public function applyTransition(string $resource, string $id, string $transition): void
+    {
+        $this->client->request(
+            'PATCH',
+            sprintf('/new-api/%s/%s/apply_transition/%s', $resource, $id, $transition),
+            [], [],
+            ['CONTENT_TYPE' => 'application/merge-patch+json', 'HTTP_ACCEPT' => 'application/ld+json'],
+            '{}'
```

Comment:
In this PATCH action, we don't provide any request's content but we call controller.
But we need to provide empty array in this argument because without it we get a syntax error